nmpc_controller/src/nmpc_controller.cpp

```diff
@@ -151,58 +151,13 @@ bool NMPCController::computeCentralizedTailPlan(const Eigen::VectorXd &initial_s
                                                 const std::vector<std::vector<bool>> &contact_schedule,
                                                 const Eigen::VectorXd &tail_initial_state,
                                                 const Eigen::MatrixXd &tail_ref_traj,
+                                                const Eigen::VectorXd &ref_ground_height,
                                                 Eigen::MatrixXd &state_traj,
                                                 Eigen::MatrixXd &control_traj,
                                                 Eigen::MatrixXd &tail_state_traj,
                                                 Eigen::MatrixXd &tail_control_traj)
 {
-  Eigen::MatrixXd ref_traj_with_tail(N_ + 1, 16),
-      state_traj_with_tail(N_ + 1, 16),
-      control_traj_with_tail(N_, 14);
-  Eigen::VectorXd initial_state_with_tail(16);
-
-  ref_traj_with_tail.setZero();
-  ref_traj_with_tail.leftCols(6) = ref_traj.leftCols(6);
-  ref_traj_with_tail.block(0, 6, N_ + 1, 2) = tail_ref_traj.leftCols(2);
-  ref_traj_with_tail.block(0, 8, N_ + 1, 6) = ref_traj.rightCols(6);
-  ref_traj_with_tail.block(0, 14, N_ + 1, 2) = tail_ref_traj.rightCols(2);
-
-  initial_state_with_tail.setZero();
-  initial_state_with_tail.head(6) = initial_state.head(6);
-  initial_state_with_tail.segment(6, 2) = tail_initial_state.head(2);
-  initial_state_with_tail.segment(8, 6) = initial_state.tail(6);
-  initial_state_with_tail.segment(14, 2) = tail_initial_state.tail(2);
-
-  mynlp_->update_solver(
-      initial_state_with_tail,
-      ref_traj_with_tail.bottomRows(N_),
-      foot_positions,
-      contact_schedule);
-  mynlp_->shift_initial_guess();
-
-  bool success = this->computePlan(initial_state_with_tail,
-                                   ref_traj_with_tail,
-                                   foot_positions,
-                                   contact_schedule,
-                                   state_traj_with_tail,
-                                   control_traj_with_tail);
-
-  state_traj = Eigen::MatrixXd::Zero(N_ + 1, 12);
-  control_traj = Eigen::MatrixXd::Zero(N_, 12);
-  tail_state_traj = Eigen::MatrixXd::Zero(N_ + 1, 4);
-  tail_control_traj = Eigen::MatrixXd::Zero(N_, 2);
-
-  state_traj.leftCols(6) = state_traj_with_tail.leftCols(6);
-  state_traj.rightCols(6) = state_traj_with_tail.block(0, 8, N_ + 1, 6);
-
-  control_traj = control_traj_with_tail.rightCols(12);
-
-  tail_state_traj.leftCols(2) = state_traj_with_tail.block(0, 6, N_ + 1, 2);
-  tail_state_traj.rightCols(2) = state_traj_with_tail.block(0, 14, N_ + 1, 2);
-
-  tail_control_traj = control_traj_with_tail.leftCols(2);
-
-  return success;
+  return true;
```

Comment:
What is this function doing now?

Reply:
It used to be the tail interface. Since we won't be using it in the main branch, I keep the name here but remove all calls. This way if we change the function format in the future, it won't cause any compilation or testing issues.